cmake/CMakeLists.txt

```diff
@@ -1048,6 +1062,12 @@ function(onnxruntime_set_compile_flags target_name)
       target_compile_definitions(${target_name} PRIVATE USE_KLEIDIAI)
     endif()
 
+    if(HAS_ARM64_SVE)
+      if("${target_name}" STREQUAL "onnxruntime_mlas")
```

Comment:
I think the preferred place to call `target_compile_definitions(onnxruntime_mlas ...)` would be in onnxruntime_mlas.cmake instead of this general helper function.